tests/symlinked_included_files.test.ts

```diff
@@ -0,0 +1,66 @@
+import { readdir } from 'fs/promises'
+import { join } from 'path'
+
+import decompress from 'decompress'
+import { dir as getTmpDir } from 'tmp-promise'
+import { expect, test } from 'vitest'
+
+import { ARCHIVE_FORMAT, zipFunction } from '../src/main.js'
+
+import { FIXTURES_ESM_DIR } from './helpers/main.js'
+
+/** Small helper function, reading a directory recursively and returning a record with the files and if it is a symlink or not */
```

Comment:
Comment looks good, but my question was about whether this should be moved to [a helper file](https://github.com/netlify/zip-it-and-ship-it/tree/main/tests/helpers) so it becomes easier to reuse in other tests.

Reply:
I don't think we need it in other tests